hoodie-client/src/main/java/com/uber/hoodie/io/HoodieAppendHandle.java

```diff
@@ -219,6 +219,9 @@ public void close() {
       }
       writeStatus.getStat().setNumWrites(recordsWritten);
       writeStatus.getStat().setNumDeletes(recordsDeleted);
+      // an estimate of the number of bytes written
+      writeStatus.getStat().setTotalUncompressedWriteBytes(recordsWritten*averageRecordSize);
```

Comment:
really like to understand how your IDE is setup :).. how come it missed formatting the space between * in this diff, while it changed it everywhere in the other.. 

we really need to do #287